lib/cocoapods/installer.rb

```diff
@@ -399,6 +400,21 @@ def verify_no_duplicate_framework_names
       end
     end
 
+    def verify_no_duplicate_library_names
```

Comment:
@DanToml what about adding this check into `verify_no_duplicate_framework_names` and rename it `verify_no_duplicate_framework_and_library_names`? Or moving common logic into a base method and use it inside these two methods? Currently these two functions are almost the same, and could be handled together in a method, or use a common base method.